android/src/main/kotlin/com/posthog/flutter/PosthogFlutterPlugin.kt

```diff
@@ -500,6 +503,15 @@ class PosthogFlutterPlugin :
         }
     }
 
+    private fun getSessionId(result: Result) {
+        try {
+            val sessionId = PostHog.getSessionId()
+            result.success(sessionId.toString())
```

Comment:
```suggestion
            result.success(sessionId?.toString())
```

sessionId can be null here